pyDeltaRCM/model.py

```diff
@@ -128,29 +131,66 @@ def finalize(self):
         self._is_finalized = True
 
     @property
-    def time_step(self):
+    def time(self):
+        """Elapsed model time in seconds.
+        """
+        return self._time
+
+    @property
+    def dt(self):
         """The time step.
 
         Raises
         ------
         UserWarning
             If a very small timestep is configured.
         """
-        return self._time_step
+        return self._dt
 
-    @time_step.setter
-    def time_step(self, new_dt):
+    @property
+    def time_step(self):
+        """Alias for `dt`.
+        """
+        return self._dt
+
+    @dt.setter
+    def dt(self, new_dt):
         if new_dt * self.init_Np_sed < 100:
             warnings.warn(UserWarning('Using a very small timestep, '
                                       'Delta might evolve very slowly.'))
 
-        self.Np_sed = int(new_dt * self.init_Np_sed)
-        self.Np_water = int(new_dt * self.init_Np_water)
+        # self.Np_sed = int(new_dt * self.init_Np_sed)
```

Comment:
This is one place I had a question, @elbeejay 

Do you know why the number of parcels would be recalculated based on the timestep? Leaving this on, at least with the current config, kind of blows things up. I'm not sure why you would want to change the parcel number anyway...

Reply:
Because the timestep size is established based on the input sediment supply, I believe that by doing this parcel number scaling, the idea was to reduce the computational time. So in this way the timestep is used to reduce the number of parcels being routed based on the input sediment supply (as opposed to setting some kind of arbitrary cutoffs for rounding off the number of parcels).

Truth be told, I think that changing the number of parcels internally like this takes away a lot of power from the user to control that input. We know that routing too few parcels will create unrealistic delta shapes -- so I am all for removing this dynamic adjustment of the number of parcels in favor of giving the user complete control here.

For what it is worth, this dynamic parcel adjustment was not present in the original Matlab DeltaRCM implementation...

Reply:
removed the parcel number change routine.